lib/syncUserFollowers.js

```diff
@@ -0,0 +1,124 @@
+/*global Buffer*/
+'use strict';
+
+/**
+ * syncUserFollows API client
+ * Abstraction over the myFT API (v3)
+ * Futher abstration over our myFTClient for sync follower functionality
+ */
+
+const log = require('@financial-times/n-logger').default;
+const config = require('./helpers/config');
+const clientErrors = require('./clientErrors');
+const myFT = require('./myFTClient');
+const kinesis = require('./kinesisClient');
+
+const relationshipProperties = {
+	byTool: config.FT_TOOL_ID,
+	byUser: config.FT_TOOL_ADMIN_ID
```

Comment:
Can we tke these out of `config` and just hardcode them?

Reply:
I'd rather keep them there just bcause they are use in syncUserFollwers and myfClient